app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js

```diff
@@ -297,9 +302,22 @@ class MigrationsCompletedList extends React.Component {
                                 >
                                   {__('Archive')}
                                 </MenuItem>
-                              </DropdownKebab>
-                            </div>
-                          )
+                              )}
+                              <DeleteMigrationMenuItem
```

Comment:
Can we use the new `StopPropagationOnClick` component here as well?

Reply:
Ah, good call.

Reply:
Was there a particular reason you implemented `<StopPropagationOnClick>` differently here than [here](https://github.com/ManageIQ/manageiq-v2v/pull/626/files#diff-1475e5bbe3c2cf1ac448c09f7ecf928bR125)?

It might be nice to not have to introduce [this conditional](https://github.com/ManageIQ/manageiq-v2v/pull/626/files#diff-4802a1e9ccfaa210190b7623d319b9c5R14)

Reply:
I only did that to not disrupt the existing structure too much. We could make them consistent, but i figured it might be useful to support both div and span there.

Reply:
I think it makes sense to refactor for consistency's sake.  Unless it breaks something, lol

Reply:
Sounds good. Refactored and pushed.